homeassistant/components/bond/entity.py

```diff
@@ -7,7 +8,7 @@
 from .utils import BondDevice, BondHub
 
 
-class BondEntity:
+class BondEntity(ABC):
```

Comment:
This class should also inherit `Entity` since it's using the entity api.

Reply:
thanks, updated